students/EricAdams/Iterators_and_Generators_Packaging/generator_lab/generator_solution.py

```diff
@@ -0,0 +1,74 @@
+"""
+generator_solution.py
+
+Solutions for the generator lab excercises.
+
+"""
+
+
+def intsum():
+    '''sum using generator
+    '''
+    x = list(range(0, 1000))
+    total = 0
+    for y in x:
+        total = total + y
+        yield total
+
+
+def doubler():
+    '''Double the previous value using a generator
+    '''
+    x = list(range(1, 1000))
+    for i, y in enumerate(x):
```

Comment:
same thing here, you can do:

for i, y in enumerate(range(1, 1000)):

though not much need for enumerate, either -- why not just:

for i in range(1000)

Also -- no need for having a terminating value, but if you do, better to pass it in to the generator function.